java/src/org/openqa/selenium/devtools/BUILD.bazel

```diff
@@ -65,6 +65,7 @@ java_library(
         ":mutation-listener",
     ],
     visibility = [
+        "//java/src/org/openqa/selenium/grid:__pkg__",
```

Comment:
This will be a problem, causing the classes in the `org.openqa.selenium.grid` package to not only be in the `org.seleniumhq.selenium:selenium-remote-driver` maven artifact, but also in `org.seleniumhq.selenium:selenium-grid` maven artifact. This is one of the reasons why we try and keep `visibility` as close to empty as possible :) 

If we really want to avoid having the CDP classes in the server, then we can create a new `java_export` target for `//java/src/org/openqa/selenium/devtools`, mark that as being publicly visible, and then have both the remote and grid packages depend upon it.

Reply:
And how about just adding a `//java/src/org/openqa/selenium/remote` runtime_dep in `//java/src/org/openqa/selenium/grid`?
We have it in the server anyway due to transitive dependencies e.g. by the `//java/src/org/openqa/selenium/chrome`.

So there is no need to modify the visibility of `//java/src/org/openqa/selenium/devtools`?